app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError, OutdatedVaccineError, NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+
+        if ("vaccine" not in visitor
+                or "expiration_date" not in visitor["vaccine"]):
+            raise NotVaccinatedError("Visitor is not vaccinated!")
+
+        vac_expire = visitor["vaccine"]["expiration_date"]
```

Comment:
The `expiration_date` should be converted to a `datetime.date` object before comparison. Currently, it is likely a string, which will cause a TypeError when compared with `today`.